e2e/scripts/st_file_uploader.py

```diff
@@ -14,14 +14,17 @@
 
 import streamlit as st
 
-
+st.set_option("deprecation.v00_064_showfileUploaderEncoding", True)
 result = st.file_uploader("Drop a file:", type=["txt"])
 # result = st.file_uploader("Drop a file:", type=["txt"], accept_multiple_files=False)
 if result is not None:
     st.text(result.getvalue())
 else:
     st.text("No upload")
 
+st.set_option("deprecation.v00_064_showfileUploaderEncoding", False)
```

Comment:
I left a nit-picky comment for Adrien about this format [over here](https://www.notion.so/streamlit/Let-s-talk-about-deprecation-warnings-67885d263aa14bb1af59b497e6880a07#8e24aad69f564301a71e41b34887835f) and pinged him on Slack. I'll make sure we have a response in the next couple of hours, so please wait before merging.

Reply:
Responded and updating